chart/tests/test_annotations.py

```diff
@@ -0,0 +1,129 @@
+    "statsd",
+    "jobs",
+    "redis",
+)
+
+
+class AnnotationsTest(unittest.TestCase):
+    def test_service_account_annotations(self):
+        k8s_objects = render_chart(
+            values={
+                "cleanup": {
+                    "enabled": True,
+                    "serviceAccount": {
+                        "annotations": {
+                            "example": CUSTOM_CLEANUP_ANNOTATION,
+                        },
+                    },
+                },
+                "scheduler": {
+                    "serviceAccount": {
+                        "annotations": {
+                            "example": CUSTOM_SCHEDULER_ANNOTATION,
+                        },
+                    },
+                },
+                "webserver": {
+                    "serviceAccount": {
+                        "annotations": {
+                            "example": CUSTOM_WEBSERVER_ANNOTATION,
+                        },
+                    },
+                },
+                "workers": {
+                    "serviceAccount": {
+                        "annotations": {
+                            "example": CUSTOM_WORKER_ANNOTATION,
+                        },
+                    },
+                },
+                "flower": {
+                    "serviceAccount": {
+                        "annotations": {
+                            "example": CUSTOM_FLOWER_ANNOTATION,
+                        },
+                    },
+                },
+                "statsd": {
+                    "serviceAccount": {
+                        "annotations": {
+                            "example": CUSTOM_STATSD_ANNOTATION,
+                        },
+                    },
+                },
+                "redis": {
+                    "serviceAccount": {
+                        "annotations": {
+                            "example": CUSTOM_REDIS_ANNOTATION,
+                        },
+                    },
+                },
+                "pgbouncer": {
+                    "enabled": True,
+                    "serviceAccount": {
+                        "annotations": {
+                            "example": CUSTOM_PGBOUNCER_ANNOTATION,
+                        },
+                    },
+                },
+                "jobs": {
+                    "serviceAccount": {
+                        "annotations": {
+                            "example": CUSTOM_JOBS_ANNOTATION,
+                        },
+                    },
+                },
+                "executor": "CeleryExecutor",  # create worker deployment
+            },
+        )
+
+        list_of_annotation_values_in_objects = [
+            k8s_object['metadata']['annotations']['example']
+            for k8s_object in k8s_objects
+            if k8s_object['kind'] == "ServiceAccount"
+        ]
+
+        self.assertCountEqual(
+            list_of_annotation_values_in_objects,
+            CUSTOM_ANNOTATION_VALUES,
+        )
```

Comment:
@DerekHeldtWerle In addition to count, can we also assert that the values match?

Reply:
@ianstanton [assertCountEqual](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual) actually does exactly what you've described.